Sources/Controllers/MessagesViewController.swift

```diff
@@ -525,6 +525,11 @@ UICollectionViewDelegateFlowLayout, UICollectionViewDataSource, UIGestureRecogni
         let velocity = panGesture.velocity(in: messagesCollectionView)
         return abs(velocity.x) > abs(velocity.y)
     }
+    
+    open func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer) -> Bool {
```

Comment:
What exactly this method does btw. ? :)

Reply:
For MessageKit it self, nothing changes because the default return of this method is true. I just add this, because im trying to implement some feature with multiple touch events and this method kind of help me. :)

Reply:
Well, this is not even override, is it? So what would change if you return false here, if its not used?